src/System.ValueTuple/src/System/ValueTuple/ValueTuple.cs

```diff
@@ -405,7 +411,12 @@ int ITupleInternal.GetHashCode(IEqualityComparer comparer)
         /// </remarks>
         public override string ToString()
         {
-            return "(" + Item1 + ")";
+            return "(" + Item1.ToString() + ")";
```

Comment:
@stephentoub, I thought string interpolation outperforms string concatenation. Apparently, it performs rather poorly (3x slower). I created a quick test: https://gist.github.com/jasonwilliams200OK/5395c3ea4c28b5e4673d28c424aed272, with `j = $"{j} : {i.ToString()}"`, it takes about 600ms and while `j = j + ":" + i.ToString();` gives ~180ms on my machine.

Is this a known fact that to get better performance, we should use concatenation and to save allocations, use interpolation (at the cost of performance)? And to get best of both worlds, use `StringBuilder`.